### ISSUE TITLE
Request: Adds an overloaded method for QueryAll<TEntity> operation that takes tableName parameter

### ISSUE THREAD

**Issue by BieleckiLtd** [mikependon/RepoDB#521]
```
var users = connection.QueryAll(tableName: "[dbo].[Users]", cache: Cache, cacheKey: "AllUsers");
var users = connection.QueryAll<User>(tableName: "[dbo].[Users]", cache: Cache, cacheKey: "AllUsers"); // no overload :(
```

**Comment by mikependon** [comment]
Hey, the code below is not release yet. Are you currently using the code line from the Main branch when you logged this request?

```csharp
var users = connection.QueryAll<User>(tableName: "[dbo].[Users]", cache: Cache, cacheKey: "AllUsers");
```

**Comment by mikependon** [comment]
And besides, the one in the main branch that is up for release for F# community has it supported already.
Please see the code [line](https://github.com/mikependon/RepoDb/blob/b7618a1454bad17207ff0a1bf13a45ed97bfee67/RepoDb.Core/RepoDb/Operations/DbConnection/QueryAll.cs#L43) here.

**Comment by mikependon** [comment]
Upon analyzing it further, the timing of this this request might be ahead from our planned upcoming release which could support this. We are now working on this overload already for almost 2 weeks now.

Please see this story #499. 

This means that this feature will be available on the next beta release.

**Comment by BieleckiLtd** [comment]
Nice one! Many thanks for a quick feedback. I am currently using latest released NuGet package (1.11.6)

**Comment by mikependon** [comment]
@BieleckiLtd - The fix is now available at [RepoDB v1.12.0-beta1](https://www.nuget.org/packages/RepoDb/1.12.0-beta1) and [RepoDb.SqlServer v1.1.0-beta1](https://www.nuget.org/packages/RepoDb.SqlServer/1.1.0-beta1).

**Comment by mikependon** [comment]
Closing this ticket now. Please do let us know if you have any further questions. Thanks
